t/plugin/basic-auth.t

```diff
@@ -20,6 +20,46 @@ repeat_each(2);
 no_long_string();
 no_root_location();
 no_shuffle();
+log_level('info');
+worker_connections(1024);
+
+add_block_preprocessor(sub {
+    my ($block) = @_;
+
+    my $http_config = $block->http_config // <<_EOC_;
```

Comment:
We already have an upstream for this, see: https://github.com/apache/apisix/blob/70174d58cd4248f55dada5c2bc7396cdf73972c7/t/lib/server.lua#L392

Use /echo is enough.

Reply:
Done.